pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -85,11 +85,11 @@ func (q *graceTerminateRSList) flushList(handler func(rsToDelete *listItem) (boo
 	for name, rs := range q.list {
 		deleted, err := handler(rs)
 		if err != nil {
-			klog.Errorf("Try delete rs %q err: %v", name, err)
+			klog.ErrorS(err, "Try delete rs err", "name", name)
 			success = false
 		}
 		if deleted {
-			klog.Infof("lw: remote out of the list: %s", name)
+			klog.InfoS("lw: remote out of the list", "name", name)
```

Comment:
A lot of these messages are just terrible. It seems like someone who understands the code needs to go through and make them more sane...

Reply:
I've gone through the code and making necessary changes